gcp/table_gcp_kubernetes_cluster.go

```diff
@@ -0,0 +1,424 @@
+				Description: ColumnDescriptionAkas,
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromP(gcpKubernetesClusterTurbotData, "Akas"),
+			},
+
+			// standard gcp columns
+			{
+				Name:        "location",
+				Description: ColumnDescriptionLocation,
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "project",
+				Description: ColumnDescriptionProject,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromP(gcpKubernetesClusterTurbotData, "Project"),
+			},
+		},
+	}
+}
+
+//// LIST FUNCTION
+
+func listKubernetesClusters(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	plugin.Logger(ctx).Trace("listKubernetesClusters")
+
+	// Create Service Connection
+	service, err := ContainerService(ctx, d)
+	if err != nil {
+		return nil, err
+	}
+
+	// Get project details
+	projectData, err := activeProject(ctx, d)
+	if err != nil {
+		return nil, err
+	}
+	project := projectData.Project
+
+	resp, err := service.Projects.Locations.Clusters.List("projects/" + project + "/locations/-").Do()
+	if err != nil {
+		return nil, err
+	}
+
+	for _, cluster := range resp.Clusters {
+		d.StreamListItem(ctx, cluster)
+	}
+
+	return nil, nil
+}
+
+func getKubernetesCluster(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	plugin.Logger(ctx).Trace("getKubernetesCluster")
+
+	// Create Service Connection
+	service, err := ContainerService(ctx, d)
+	if err != nil {
+		return nil, err
+	}
+
+	// Get project details
+	projectData, err := activeProject(ctx, d)
+	if err != nil {
+		return nil, err
+	}
+	project := projectData.Project
+
+	name := d.KeyColumnQuals["name"].GetStringValue()
+	location := d.KeyColumnQuals["location"].GetStringValue()
+
+	resp, err := service.Projects.Locations.Clusters.Get("projects/" + project + "/locations/" + location + "/clusters/" + name).Do()
+	if err != nil {
+		return nil, err
+	}
+
+	return resp, nil
+}
+
+//// TRANSFORM FUNCTION
+
+func gcpKubernetesClusterTurbotData(ctx context.Context, d *transform.TransformData) (interface{}, error) {
+	plugin.Logger(ctx).Trace("gcpKubernetesClusterTurbotData")
+	cluster := d.HydrateItem.(*container.Cluster)
+
+	splitName := strings.Split(cluster.SelfLink, "/")
+	akas := []string{strings.Replace(cluster.SelfLink, "https", "gcp", 1)}
+
+	if d.Param.(string) == "ClusterName" {
+		return splitName[9], nil
+	} else if d.Param.(string) == "Location" {
+		return splitName[7], nil
+	} else if d.Param.(string) == "Project" {
+		return splitName[5], nil
+	} else {
+		return akas, nil
+	}
+}
+
+func gcpKubernetesClusterLocationType(ctx context.Context, d *transform.TransformData) (interface{}, error) {
+	plugin.Logger(ctx).Trace("gcpKubernetesClusterLocationType")
+	cluster := d.HydrateItem.(*container.Cluster)
+
+	splitName := strings.Split(cluster.SelfLink, "/")
+
+	if splitName[6] == "locations" {
+		return "Regional", nil
+	} else {
+		return "Zonal", nil
+	}
```

Comment:
No need to add `else`.
```suggestion
	} 
	return "Zonal", nil
```

Reply:
@rajeshbal65 I think we should change the value to `REGIONAL/ZONAL`.

Reply:
In gcp console It is `Regional` nas `Zonal`

![Screenshot 2021-05-24 at 1 57 16 PM](https://user-images.githubusercontent.com/46851617/119319405-08958680-bc98-11eb-85b1-918b98231b84.png)